Libraries/ReactIOS/YellowBox.js

```diff
@@ -82,9 +84,27 @@ function updateWarningMap(format, ...args): void {
     ...args.slice(argCount).map(stringifySafe),
   ].join(' ');
 
-  const count = _warningMap.has(warning) ? _warningMap.get(warning) : 0;
-  _warningMap.set(warning, count + 1);
+  var warningInfo = _warningMap.get(warning);
+  if (warningInfo) {
+    warningInfo.count += 1;
+  } else {
+    warningInfo = {count: 1, stacktrace: []};
+  }
+
+  _warningMap.set(warning, warningInfo);
   _warningEmitter.emit('warning', _warningMap);
+
+  var error = new Error();
```

Comment:
like in tests https://github.com/facebook/react-native/blob/master/Libraries/JavaScriptAppEngine/Initialization/__tests__/parseErrorStack-test.js#L34